bin/launch_interpret_json.py

```diff
@@ -30,11 +31,18 @@ def interpret_json(input_json: dict) -> list:
     #print("\nnoise_configurations :\n", schema.noise_arg, "\n", type(schema.noise_arg))
     #print("\nsplit_configurations :\n", schema.split_arg, "\n", type(schema.split_arg))
     #print("\ncustom_configurations :\n", schema.custom_arg, "\n", type(schema.custom_arg))
-    print(schema.number_culumn_wise_val)
+    #print(schema.number_culumn_wise_val)
     #print(schema.experiment, schema.interpret_params_mode, schema.column_names)
-    
 
```

Comment:
Ideally, prints should not be used and replaced by a logger if you are using those to debug (see https://docs.python.org/3/howto/logging.html )